### PR TITLE
Updated day 4 content

### PR DESCRIPTION
**Pull request by wisegb**
Added the following:
- Data Analysis
- Software Testing
- Example unit tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ and cover the basics of the Python language.
 ## Day 4
 
 1. Git
-2. Data Analysis
-3. Tests
+2. [Data Analysis](Day4/DataAnalysis.md)
+3. [Software Testing](Day4/Testeing.md)
```
Comment (graichen) [inline README.md]: This has a typo.

File: Day4/DataAnalysis.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    ax.set_title('Min and Max Temperature by Station')
+    ax.set_xticks(ind + width / 2)
+    ax.set_xticklabels(array_STATION)
+    ax.legend((rects1[0], rects2[0]), ('Min', 'Max'))
+    plt.show()
+```
+
+The resulting chart:
+
+![](.DataAnalysis_images\4cceee54.png)
```
Comment (graichen) [inline Day4/DataAnalysis.md]: Use forward slashes for github.

File: Day4/DataAnalysis.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
+[< Previous (Git)](Git.md) | [Day4](../README.md)| [Next (Tests) >](Tests.md) |
```
Comment (graichen) [inline Day4/DataAnalysis.md]: link to test is Testing and Testing.md

**Review by graichen — CHANGES_REQUESTED** [review]
Just a few minor edits, then looks good.

**Comment by wisegb** [comment]
@graichen Thanks for the review; corrections have been committed

**Review by graichen — APPROVED** [review]

**Pull request merged**